Mathlib/Algebra/Category/FiniteGrp/Basic.lean

```diff
@@ -0,0 +1,85 @@
+/-
+Copyright (c) 2024 Jujian Zhang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jujian Zhang, Nailin Guan, Yuyang Zhao
+-/
+import Mathlib.Algebra.Category.Grp.Basic
+
+/-!
+
+## Main definitions and results
+
+* `FiniteGrp` is the type of finite groups.
+
+-/
+
+universe u v
+
+open CategoryTheory
+
+/-- The category of finite groups -/
+@[pp_with_univ]
+structure FiniteGrp where
+  /-- A group that is finite -/
+  toGrp : Grp
+  [isFinite : Finite toGrp]
+
+/-- The category of finite groups -/
```

Comment:
```suggestion
/-- The category of finite additive groups -/
```
Have you tried using the `@to_additive` attribute?

Reply:
The to_additive failed at this and profinitegrp, I don't know exactly why.

Reply:
Let me try to have a look